Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m

```diff
@@ -94,6 +101,16 @@ - (NSString *)apiKey {
   return self.settings[@"apiKey"];
 }
 
+- (NSNumber *)delayLoad {
+    return (NSNumber *)[self.settings objectForKey:@"delayLoad"];
+}
+
+- (BOOL)shakeMenu {
```

Comment:
Hey @f2prateek – we actually need shakeMenu to be an NSNumber with a value of @0 or @1. This will not work with our SDK. It looks like I may have sent the commit to my branch too late :(